code/src/components/TaskItems.css

```diff
@@ -0,0 +1,91 @@
+.list-task {
+  display: grid;
+  grid-template-columns: 0.6fr 0.4fr;
+  background-color: rgba(206, 177, 119, 0.3); 
+  margin: 5px;
+}
```

Comment:
I think that `display: grid` would be perfect for the ul element, and then you could add a `row-gap` of 5px instead of the margin here.